apps/web/src/utils/experienceUtils.tsx

```diff
@@ -528,6 +528,11 @@ export const isPersonalExperience = (
 export const isWorkExperience = (
   e: SimpleAnyExperience,
 ): e is Omit<WorkExperience, "user"> => e.__typename === "WorkExperience";
+export const isGovWorkExperience = (
+  e: SimpleAnyExperience,
+): e is Omit<WorkExperience, "user"> =>
```

Comment:
I like this function but having it as a typeguard is a bit weird since there's no "GovWorkExperience" type in our schema that we're narrowing to.  I suggest removing the `e is Omit<WorkExperience, "user">` bit.

Reply:
That does solve the typing error .. but that check is already inside the govWorkExperience

Reply:
Ok I figure that I need this type guard in here `e is Omit<WorkExperience, "user">` so that it return as WorkExperience. Otherwise we would need that duplicate check for isWorkExperience just to return that type. 
Putting that back bit back.